### DISCUSSION TITLE
请问如何配置远程的TVbox订阅

### DISCUSSION THREAD

**Discussion by kokojacket** [monlor/docker-xiaoya#195] [Q&A]
原版的是通过修改docker_address.txt为域名+端口再重启来实现远程订阅xiaoya。
请问这个是应该怎么配置呢，看讨论里说，重启后会重置docker_address.txt.
是 开启tvbox随机订阅地址 这个吗？

**Comment by monlor** [comment]
文档里写了访问链接，直接用 不需要手动配置

**Reply by kokojacket** [reply]
外网有公网IP情况下，应该如何链接？局域网我清楚

**Reply by kokojacket** [reply]
![Clip_2024-07-11_20-45-27](https://github.com/monlor/docker-xiaoya/assets/74887370/2ef6c289-9a09-4242-98c2-7fb18444e7cd)

**Reply by monlor** [reply]
你可以弄域名 做dns解析，保证你的内外网域名一致
